### PR TITLE
metrics: Add snapshot generation wait metric

### LINKED ISSUE #14940: Add snapshot generation wait time metric

### PR DESCRIPTION
**Pull request by v01dstar**
### What is changed and how it works?

Issue Number: Close #14940

What's Changed:

```commit-message
Add snapshot generation wait duration metric
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`: n/a
- Need to cherry-pick to the release branch n/a

### Check List 

Tests 

- No code

Side effects

N/A

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by Connor1996 — COMMENTED** [review]
Please update grafana json as well

**Comment by v01dstar** [comment]
> Please update grafana json as well

Done, I imported the latest `tikv_details.json` into the grafana (v 7.5.10) dashboard in `tiup playground`. And added a panel for the new metric, then export the dashboard to a json.

I took a glance at the generated json, there are several types of changes:

* Deprecated field, e.g. `hash_key`.
* Fields reorder, this contributes to the most of the changes.
* Version, from 7.5.11 to 7.5.10, since the latter one is used in 7.1.0 TiDB release
* Relevant changes, "snapshot generation wait duration"

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "__inputs": [
     {
       "name": "DS_TEST-CLUSTER",
-      "label": "test-cluster",
+      "label": "Test-Cluster",
```
Comment (Connor1996) [inline metrics/grafana/tikv_details.json:5]: revert it

Comment (v01dstar) [reply metrics/grafana/tikv_details.json:5]: Done

File: metrics/grafana/tikv_details.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "7.5.11"
+      "version": "7.5.10"
```
[the remark this answers is not in the document]
Comment (v01dstar) [reply metrics/grafana/tikv_details.json:17]: This is version being used by latest release version in TiUP

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by bufferflies — APPROVED** [review]

**Pull request merged**